#!/usr/bin/env python
PACKAGE = "fake_imu"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("sensor_offset", double_t, 0, "Sensor Offset", 270.0, 0, 500)
gen.add("sensor_scale", double_t, 0, "Sensor Scale", -1.0/18, -180, 180)

exit(gen.generate(PACKAGE, "fake_imu", "depth"))
